libs/solid/skeleton/src/lib/components/main-menu/main-menu.component.ts

```diff
@@ -47,4 +47,5 @@ export class MainMenuComponent implements OnInit {
   private GetSlideshowCategories() {
     return new GetSlideshowCategories();
   }
+
```

Comment:
Solche kleine Änderungen sollten wir vermeiden. In diesem Fall ist es nicht so schlimm, weil es die Lesbarkeit verbessert.
Wenn es aber sonst nichts Neues im File gibt, dann ist es besser, gar keine Änderungen zu machen.